tests/functional/test_install_vcs_svn.py

```diff
@@ -21,10 +23,12 @@ def test_obtain_should_recognize_auth_info_url(call_subprocess_mock, script):
 @pytest.mark.network
 def test_export_should_recognize_auth_info_url(call_subprocess_mock, script):
     url = 'svn+http://username:password@svn.example.com/'
-    svn = Subversion()
+    # use_interactive is set to False to test that remote call options are
+    # properly added.
+    svn = Subversion(use_interactive=False)
     svn.export(script.scratch_path / 'test', url=url)
     assert call_subprocess_mock.call_args[0][0] == [
-        svn.name, 'export', '--username', 'username', '--password',
-        'password', 'http://svn.example.com/',
+        svn.name, 'export', '--non-interactive', '--username', 'username',
+        '--password', 'password', 'http://svn.example.com/',
```

Comment:
My only other comment on this PR is to add a test like this for the other two methods to ensure that the `get_remote_call_options()` are getting added for all four commands. Some suggestions for this--

1. It would be good to pull into a helper function the code common to the four cases. This way you can reduce boilerplate.
2. Since the SVN call is mocked, isn't it the case that these tests don't actually hit the network? If that's the case, the `@pytest.mark.network` decorators aren't needed.

Reply:
> Since the SVN call is mocked, isn't it the case that these tests don't actually hit the network?

Huh, yeah looks like this was erroneously added in c5024dc851fc8c5e688607f40c3a4a849617d563.

Reply:
> It would be good to pull into a helper function the code common to the four cases. This way you can reduce boilerplate.

I started to look at this. I agree there is some boilerplate, but it didn't seem too bad compared to normal testing boilerplate. Also since each test calls a different method, I couldn't come up with a way to reduce boilerplate meaningfully without making the tests harder to understand.

Could you review the latest PR and confirm you'd like this to be done (and perhaps how you think best to do it, if so)? Thanks.

Reply:
The way I would do it is make a `TestSubversionArgs` `TestCase` class, and then use the approach at the end of this section to do the patching in `setUp()`: https://docs.python.org/3.5/library/unittest.mock-examples.html#applying-the-same-patch-to-every-test-method

Then I would add two helper methods to use at the beginning and end of each test:

1. `svn, dest, url = self.get_test_data()`
2. `assert_call_args(expected)`